addons/rating/models/rating.py

```diff
@@ -53,6 +53,7 @@ def _default_access_token(self):
         'mail.message', string="Linked message",
         index=True, ondelete='cascade',
         help="Associated message when posting a review. Mainly used in website addons.")
+    is_internal = fields.Boolean('Employee Only')
```

Comment:
So related to previous comment, instead of being a related field to message, it is now its own field on rating?

Is there a reason to unsync `is_internal` from the rating compared to its message?